docs/documentation/upgrading/topics/changes/changes-26_2_0.adoc

```diff
@@ -0,0 +1,5 @@
+= Removal of the `X-XSS-Protection` header
```

Comment:
Is it possible to add this to the release notes for 26.2.0 instead of upgrading guide? AFAIK upgrading guide may contain only things, which somehow affect backwards compatibility and require some action from the Keycloak administrators/users. But this is not the case if I understand correctly?

Reply:
This still might affect some users if they re-use this header, I think the changelog is the perfect place for it. Putting it in the release notes would just clutter them, I'd rather reserve it for a nice overview of features.

Reply:
Resolving this as per https://github.com/keycloak/keycloak/pull/36881#issuecomment-2666546172